etl/README.md

```diff
@@ -0,0 +1,21 @@
+# ETL
+
+<blockquote>
+<p>ETL refers to the general process of:</p>
+
+<p>
+    <ol>
+        <li>taking raw <strong>source data</strong> (<em>"Extract"</em>)</li>
+        <li>doing some stuff to get the data in shape, possibly involving intermediate <strong>derived files</strong> (<em>"Transform"</em>)</li>
+        <li>producing <strong>final output</strong> in a more usable form (for <em>"Loading"</em> into something that consumes the data - be it an app, a system, a visualization, etc.)</li>
+    </ol>
+</p>
+</blockquote>
```

Comment:
This works, but is there a reason you chose to use HTML tags instead of the Markdown quote operator `>`?

Reply:
Also, is there a reason for this to be an un-attributed quote instead of normal body text?

Reply:
AFAIK, formatting for multi-paragraph block quotes gets messed up, and I wanted to preserve it.